djangocms_picture/templates/djangocms_picture/default/picture.html

```diff
@@ -0,0 +1,75 @@
+{% extends "djangocms_picture/base.html" %}
```

Comment:
keep in mind that this calls the filesystem on every plugin, it's not covered by our internal cms template caching system. I would avoid using a base template and instead make the default template more pluggable. Granted that it's unlikely to have 200 picture plugins on a single page, maybe for future reference.